### DISCUSSION TITLE
Idea Brainstorming

### DISCUSSION THREAD

**Discussion by avidrucker** [knowledge-for-good/knowledgeforgood#17] [Ideas]
Feel free to riff on ideas here

**Comment by avidrucker** [comment]
A11y Rainbow Communication

**Reply by avidrucker** [reply]
This idea is to communicate in an interlinked, multi-media format. Visual + auditory + textual + "smart". 

For example, it could be a 30 second video recording with audio captioning, plus an "actionable request" that is "consumable" via the receiver's favorite to-do/task list manager (Asana, JIRA, GitHub issues, Trello, Todoist, gmail, etc..)

Another example might be an emoji filter* (animated AR instead of direct video) + audio + links, where the sender and receiver both know who has seen what, and how they are responding to each thing in real time, such as person A sending a video to person B to which person A reacts to in real time.

*See either https://www.imore.com/how-use-animoji-stickers-live-photos-facetime or https://www.pocket-lint.com/apps/news/apple/142230-what-are-animoji-how-to-use-apple-s-animated-emoji

**Comment by avidrucker** [comment]
Requester service that allows users to use whatever service they want to delegate and consume tasks

**Reply by avidrucker** [reply]
For example, of the services Asana, PivotalTracker, GitHub, JIRA, etc. tasks can be assigned. On the flip side, many people like their own to-do/task list manager apps, such as Todoist, Any.Todo, and more. This service / product would allow one person to use app X to distribute/delegate tasks, while another person can use app Y to consume/complete tasks, without any friction/frustration between the two people and two apps for using different apps.

**Comment by seekanddefine** [comment]
Design a tool that has learning management built in a way to overlay your independent learning with a specifically chosen structure. It may begin with a digital literacy/reading/background knowledge evaluation and from there puts learners in a variety of tracks like (beginner, expert). Each layer of learning tracks time spent in key resources and supplemental resources. We could build in community, accountability, and motivation. After learning something the path the learner took would be displayed visually as a interactive structure/tree. Avi learned Clojure for example. He used three primary resources and tracked the hours that he spent with each one as well as feedback (the review is super important for knowledge retention.) We could build in spaced repetition and other important cognitive tools that are based on learning research. After you are done you could display the entire path that you created (much like how a homeschool student has to document their own "transcript" in order to apply for college). This could be a resource that you put on your resume to show how much you have done in a particular skill, but it could also be freely available in the community so that when Linnea, for example, jumps in to learn Clojure, they can see from the learners before them which resources were most highly recommended, the time spent on them, and better map their own learning path. Perhaps the path starts by helping the student create an ideal learning week where they put in their personal responsibilities and then the program suggest a time schedule that is ideal for learning built around their personal schedule. (I developed a paper version of this second part when I directed an Academic Success Center).
